src/test/java/io/cdap/plugin/salesforce/etl/SalesforceBatchMultiSourceETLTest.java

```diff
@@ -163,4 +164,74 @@ public void testBlackList() throws Exception {
     Assert.assertEquals(expectedSObjects, actualSObjects);
   }
 
+  @Test
+  public void testValidateWhiteListSObjects() throws Exception {
+
```

Comment:
Looks like you're not actually testing your code change? where are you calling validateSObjects()?

Reply:
added  validateSObjects() method.